dotnet/generate-all-dockerfile.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+set -e
+
+out_dir=.
+
+devices='raspberry-pi raspberry-pi2 raspberrypi3 beaglebone edison nuc vab820-quad zc702-zynq7 odroid-c1 odroid-ux3 parallella-hdmi-resin nitrogen6x cubox-i ts4900 colibri-imx6 apalis-imx6'
```

Comment:
There are some missing devices here I think, this is the list of all supported devices https://github.com/resin-io-library/base-images/blob/master/node/generate-latest-dockerfile.sh#L47

Reply:
I will use that. It would be good to have a central `targets.sh` to include so that it can be centrally managed 👍